code/ai/aicode.cpp

```diff
@@ -13417,8 +13432,13 @@ int ai_acquire_emerge_path(object *pl_objp, int parent_objnum, int allowed_path_
 			}
 		}
 
-		// cycle through the allowed paths
-		bay_path = allowed_bay_paths[Ai_last_arrive_path % num_allowed_paths];
+		// Cyborg: fall back to standard behavior if no allowed paths to avoid divide by zero -- Coverity 1523287
```

Comment:
Within this if() block, the fact that `allowed_path_mask` is not 0 means there should be guaranteed to be at least one allowed path.  It would be good to add an `UNREACHABLE` for the 0-case.  Is `UNREACHABLE` enough to silence Coverity or is the if() needed too?

Reply:
It probably won't silence coverity, but I can always dismiss the issue after adding the `UNREACHABLE`

Reply:
Actually, with an if, it should avoid the issue.